Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3112,13 +2158,23 @@ - (void)viewDidLayoutSubviews
     [self updateLastVisibleTimestamp];
 }
 
-- (void)createScrollDownButton
+- (void)createScrollButtons
+{
+    self.scrollDownButton = [self createScrollButton:@"\uf103" selector:@selector(scrollDownButtonTapped)];
+#ifdef DEBUG
+    self.scrollUpButton = [self createScrollButton:@"\uf102" selector:@selector(scrollUpButtonTapped)];
```

Comment:
I added a debug-only scroll up button.  It'll be useful when we restore the "load more" feature.